README.md

```diff
@@ -123,5 +123,8 @@ MIT (c) 2014 Vsevolod Strukchinsky (floatdrop@gmail.com)
 [travis-url]: https://travis-ci.org/floatdrop/gulp-watch
 [travis-image]: http://img.shields.io/travis/floatdrop/gulp-watch.svg?style=flat
 
+[appveyor-url]: https://ci.appveyor.com/project/UltCombo/gulp-watch/branch/master
```

Comment:
I've created the [project on AppVeyor](https://ci.appveyor.com/project/UltCombo/gulp-watch), so although it points to the `floatdrop/gulp-watch` repository, it has my AppVeyor account name in the URL.

@floatdrop Perhaps you need to create the repository and set webhooks, I'm not sure.